src/stories/Library/content-list-item/ContentListItemStacked.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from "react";
+import { ReactComponent as ArrowSmallRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg";
+
+interface ContentListItemStackedProps {
+  href: string;
+  time: string;
+  date: string;
+}
+
+const ContentListItemStacked: FC<ContentListItemStackedProps> = ({
+  href = "#",
+  time,
+  date,
+}) => {
+  return (
+    <a
+      href={href}
+      className="content-list-item-stacked arrow__hover--right-small"
+    >
+      <time className="content-list-item-stacked__content">
```

Comment:
@kasperbirch1 has added a visually hidden title for the event in https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/785/commits/5d3dfe7db3bb945d6005a1a7393a1d8e78d776ff to improve accessibility.

Consider if this should be a part of the design system as well.